tests/testthat/test-pivot-wide.R

```diff
@@ -885,63 +939,3 @@ test_that("`id_cols` compat behavior doesn't trigger if named `...` are supplied
     pivot_wider(df, ids = id)
   })
 })
-
-# Tests for issue #1609 / #1482 -----------------------------------------------
-
-test_that("doesn't crash when `id_cols` selects column removed by `names_from` (#1609)", {
-  # Original issue scenario
-  x <- c(1, 2, 12, 31, 123, 2341)
-  df <- data.frame(x = x)
-
-  # This should produce a proper tidyselect error, not an internal error
-  expect_snapshot(error = TRUE, {
-    df %>%
-      dplyr::mutate(y = stringr::str_split(x, "")) %>%
-      unnest(cols = y) %>%
-      pivot_wider(
-        id_cols = x,
-        values_from = y,
-        names_from = x
-      )
-  })
-})
-
-test_that("doesn't crash when `id_cols` selects non-existent column (#1482)", {
-  # Related issue scenario
-  df <- tibble(name = c("x", "y"), value = c(1, 2))
-
-  # Should get tidyselect error, not internal error
-  expect_snapshot(error = TRUE, {
-    pivot_wider(
-      df,
-      id_cols = c("non", "existent"),
-      names_from = name,
-      values_from = value
-    )
-  })
-
-  # Character vector case
-  expect_snapshot(error = TRUE, {
-    pivot_wider(
-      df,
-      id_cols = c("a", "b", "c"),
-      names_from = name,
-      values_from = value
-    )
-  })
-
-  df2 <- tibble(y = c("a", "a", "b", "c"), z = c(21, 22, 23, 24))
-
-  expect_snapshot(error = TRUE, {
-    pivot_wider(
-      df,
```

Comment:
This should be using `df2`, the tests aren't useful otherwise, looks like a thinko